src/Accelerators/NNPA/Conversion/ONNXToZHigh/ONNXLegalityCheck.cpp

```diff
@@ -37,8 +37,9 @@ bool onnxToZHighUnsupportedReport(Operation *op, const std::string &message) {
 }
 
 /// Report incompatibility with NNPA Level.
-bool onnxToZHighInCompatibilityReport(Operation *op) {
-  std::string onnxMlirNnpaLevel(NNPA_Z16);
+bool onnxToZHighInCompatibilityReport(
+    Operation *op, std::string inputNNPALevel) {
+  std::string onnxMlirNnpaLevel(inputNNPALevel);
```

Comment:
It looks we can use `inputNNPALevel` directly in the `message` instead of create a new `onnxMlirNnpaLevel`.

Reply:
Good catch, updated, thanks!